### ISSUE TITLE
dashboard page stops working

### ISSUE THREAD

**Issue by ravenlord1** [bjornstar/Tumblr-Savior#52]
It seems that if the app tries blocking too many images then the dashboard page simply loads nothing.  Removing the blacklist term(s) restores the dashboard function but of course it also allows all of the images that were previously blocked. Windows 7, Waterfox 56.0.3.

**Comment by bjornstar** [comment]
Yeah, it happens on every browser when all posts are blocked. I've tried looking into this before but haven't found a nice way to automatically trigger loading more posts.

**Comment by ravenlord1** [comment]
Thanks for the reply.  I'll leave notifications enabled then, as scrolling past those is still way better than scrolling through a bunch of images  :)
